server/lib/util.js

```diff
@@ -118,3 +119,19 @@ function dateFormatter(rows, dateFormat) {
 
   return rows;
 }
+
+// this function separate amounts by curreny_id
```

Comment:
Could you add in the comment that this function requires that each row must contain : amount, currency_id and symbol